ci/jenkins/README.md

```diff
@@ -66,6 +66,9 @@ should be deleted. This ensures that all tests are run on a clean testbed.
   | :------------: | :-----------------: | :-------------: | :------: |
   | 1.16.10        |  Standard_DS2_v2    |  Ubuntu 16.04   |[![Build Status](http://jenkins.antrea-ci.rocks/buildStatus/icon?job=cloud-antrea-aks-conformance-net-policy)](http://jenkins.antrea-ci.rocks/view/cloud/job/cloud-antrea-aks-conformance-net-policy/)|
 
+* [daily-elk-flow-collector-validate](https://jenkins.antrea-ci.rocks/job/antrea-daily-elk-flow-collector-validate-for-period/):
+  daily validation of elk flow collector manifest. If build fails, Jenkins will send an email to 
+  projectantrea-dev@googlegroups.com for notification.
```

Comment:
Please add a blank line before the next line "If you need to run the K8s community tests locally...". Now they are in the same line when reading.

Reply:
Done. Thanks!